OpenBSD uses $(PREFIX)/man as a MANDIR, we use $(PREFIX)/share/man.

diff -upr docbook2mdoc-1.1.0.orig/Makefile docbook2mdoc-1.1.0/Makefile
--- docbook2mdoc-1.1.0.orig/Makefile	2020-12-29 10:24:38.687808981 +0100
+++ docbook2mdoc-1.1.0/Makefile	2020-12-29 10:25:00.454608509 +0100
@@ -20,9 +20,9 @@ www: docbook2mdoc.1.html docbook2mdoc-$(
 
 install: all
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
-	mkdir -p $(DESTDIR)$(PREFIX)/man/man1
+	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
 	install -m 0755 docbook2mdoc $(DESTDIR)$(PREFIX)/bin
-	install -m 0444 docbook2mdoc.1 $(DESTDIR)$(PREFIX)/man/man1
+	install -m 0444 docbook2mdoc.1 $(DESTDIR)$(PREFIX)/share/man/man1
 
 installwww: www
 	mkdir -p $(WWWPREFIX)/snapshots
